src/govuk/overrides/_spacing.scss

```
@@ -52,7 +56,38 @@ $_spacing-directions: (
   }
 }
 
+/// Generate static spacing override classes
+///
+/// Generate spacing override classes for the given property (e.g. margin)
+/// for each point in the non-responsive spacing scale.
+///
+/// @param {String} $property - Property to add spacing to (e.g. 'margin')
+///
+/// @example css
+///   .govuk-\!-static-margin-4 {
+///      margin: 20px !important;
+///   }
+///
+/// @access private
+@mixin _govuk-generate-static-spacing-overrides($property) {
+  @each $spacing-point in map-keys($govuk-spacing-points) {
+    .govuk-\!-#{$property}-static-#{$spacing-point} {
```

This doesn't match up with the suggested class name format. For example, it outputs `.govuk-\!-margin-static-1` rather than `.govuk-\!-static-margin-1` (as suggested it should be in the code comment above)

```suggestion
    .govuk-\!-static-#{$property}-#{$spacing-point} {
```

---

Great catch – and apologies to @patrickpatrickpatrick as that was my mistake, introduced in the branch that I pushed.